bip-0158/gentestvectors.go

```diff
@@ -15,13 +15,13 @@ import (
 	"io"
 	"io/ioutil"
 	"os"
-	"path"
+	"path/filepath"
 
-	"github.com/roasbeef/btcd/chaincfg/chainhash"
-	"github.com/roasbeef/btcd/rpcclient"
-	"github.com/roasbeef/btcd/wire"
-	"github.com/roasbeef/btcutil/gcs"
-	"github.com/roasbeef/btcutil/gcs/builder"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/rpcclient"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcutil/gcs/builder"
```

Comment:
Currently the referenced gcs builder github.com/btcsuite/btcutil/gcs/builder add the txid.
Is this the new version https://github.com/Roasbeef/btcutil/blob/gcs-modifications/gcs/builder/builder.go?

Reply:
The code used to generate these test vectors can be found here: https://github.com/Roasbeef/btcutil/commits/gcs-modifications. It'll be merged back into `btcsuite` proper once this PR lands.